src/read.c

```diff
@@ -2048,6 +2167,11 @@ static avifResult avifDecoderItemReadAndParse(const avifDecoder * decoder,
         *codecType = avifGetCodecType(item->type);
         AVIF_ASSERT_OR_RETURN(*codecType != AVIF_CODEC_TYPE_UNKNOWN);
     }
+    // TODO(yguyon): If AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM is defined, backward-incompatible
+    //               files with a primary 'sato' Sample Transform derived image item could be
+    //               handled here (compared to backward-compatible files with a 'sato' item in the
+    //               same 'altr' group as the primary regular color item which are handled in
+    //               avifDecoderDataFindSampleTransformImageItem() below).
```

Comment:
Nit: Lines 2171-2174 do not need to be aligned with the colon (:) in line 2170.

Reply:
I prefer it that way. Also some editors such as CLion only highlight the aligned part because they have no way of knowing where the TODO ends otherwise.

https://google.github.io/styleguide/cppguide.html#TODO_Comments does not cover that topic.